docs/collect-filling-forms.rst

```diff
@@ -236,27 +236,21 @@ Required Questions
 Changing language of a form
 -----------------------------
 
-If a form is available in multiple languages, you can choose a language in which you want the questions to appear. 
+If a form is available in multiple languages, you can choose a language in which you want the questions to appear. This is separate from the Collect application language which is set by the device settings by default or can be manually selected from :docs:`Settings <collect-settings>`.
 
-Open the *Action Menu* (:menuselection:`⋮`) and select :guilabel:`Change Language`.
+Open the *Action Menu* (:menuselection:`⋮`) and select :guilabel:`Change Language`. You will then see a dialog with all languages defined by the form.
 
-.. image:: /img/collect-filling-forms/question-screen-highlight-kebab.* 
-  :alt: A question screen in the Collect app. The Action Menu ("kebab") in the top-right corner is circled in red.
+.. image:: /img/collect-filling-forms/question-screen-highlight-change-language.* 
+  :alt: A question screen in the Collect app. The Action Menu ("kebab") in the top-right corner has been tapped. There's an option menu with an arrow pointing to "Change language".
```

Comment:
replace kebab with ⋮

Reply:
That's in alt text so I don't think ⋮ would be helpful.